src/components/contextual/pages/vebal/LMVoting/GaugesFilters.vue

```diff
@@ -0,0 +1,105 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+
+/**
+ * TYPES
+ */
+type Props = {
+  networkOptions: { [key: number]: string };
+  debouncedHideExpiredGauges: boolean;
+  debouncedActiveNetworkFilters: number[];
+};
+
+const emit = defineEmits<{
+  (e: 'choose-network', value: number): void;
+  (e: 'update:debouncedHideExpiredGauges', value: boolean): void;
+  (e: 'update:debouncedActiveNetworkFilters', value: number[]): void;
+}>();
+
+/**
+ * PROPS
+ */
+const props = defineProps<Props>();
+
+function handleExpInput(e) {
+  emit('update:debouncedHideExpiredGauges', e.target.checked);
+}
+
+const networkFiltersArr = ref([...props.debouncedActiveNetworkFilters]);
+function updateNetwork(network: number) {
+  const index = networkFiltersArr.value.indexOf(network);
+
+  index === -1
+    ? networkFiltersArr.value.push(network)
+    : networkFiltersArr.value.splice(index, 1);
+
+  emit('update:debouncedActiveNetworkFilters', [...networkFiltersArr.value]);
+}
+</script>
+
+<template>
+  <BalPopover noPad>
+    <template #activator>
+      <div class="gauge-filter">
+        <div class="flex flex-1 items-center h-full text-gray-600">
+          <BalIcon name="filter" size="lg" class="mr-3 dark:text-gray-100" />
+          <div class="text-gray-600 dark:text-gray-100">
+            {{ $t('gaugeFilter.moreFilters') }}
+          </div>
+        </div>
+      </div>
+    </template>
+    <template #default="{ close }">
+      <div
+        class="flex overflow-hidden flex-col py-4 px-3 w-64 rounded-lg"
+        @click="close"
+      >
```

Comment:
Please use BalBtn for this. The style might not be exactly the same as Pon's designs but we have to stay consistent with buttons.

Reply:
If @pkattera want's to change the style of buttons we have to apply that to BalBtn and therefore everywhere.